pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+**Solution Descrition:**
+
+A clear and concise solution description of the problem.
+
+**Current Behavior:**
+
+Describe with example of the current behavior of the proposed code changes.
+
+**Unittest Added?** Yes/No
```

Comment:
Probably change to "Unit tests". I like how you are getting devs to explain why they don't have any tests 😄

Reply:
lol yes